stdlib/public/stubs/LibcShims.cpp

```diff
@@ -332,44 +352,101 @@ swift::_stdlib_cxx11_mt19937_uniform(__swift_uint32_t upper_bound) {
 }
 
 #if defined(__APPLE__)
-#include <Security/Security.h>
+
 SWIFT_RUNTIME_STDLIB_INTERNAL
 void swift::_stdlib_random(void *buf, __swift_size_t nbytes) {
   if (__builtin_available(macOS 10.12, iOS 10, tvOS 10, watchOS 3, *)) {
```

Comment:
Out of curiosity, why isn't this conditional written in the form of a macro? And why not just stick with `SecRandomCopyBytes` for all Apple platforms?

Reply:
Yea so Greg Parker brought this up as well in the actual pr, but I know a few on the mailing list had said that the Darwin solution should use `arc4random`. In addition to those comments, the whole family of `arc4random` is advertised as being `always successful` on the mac man pages `man arc4random`, so removing the possibility of error in something I'm in favor of. Although, I do see using one solution for all versions as an easier implementation.

Reply:
> Out of curiosity, why isn't this conditional written in the form of a macro?

@xwu The [original](https://github.com/Azoy/swift/blob/b20b2385d878dada4770c071fa193a98d4a4eb99/stdlib/public/stubs/LibcShims.cpp#L338-L360) was using macros. I suggested [__builtin_available](https://clang.llvm.org/docs/LanguageExtensions.html#objective-c-available) because it should be a *runtime* conditional. AFAIK, until the stdlib is a system library, `MAC_OS_X_VERSION_MIN_REQUIRED` will always be `MAC_OS_X_VERSION_10_9` (see [supported target platforms](https://swift.org/download/#supported-target-platforms)).